app/containers/markdown/Preview.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { StyleProp, Text, TextStyle } from 'react-native';
+import removeMarkdown from 'remove-markdown';
+
+import shortnameToUnicode from '../../utils/shortnameToUnicode';
+import { themes } from '../../constants/colors';
+import { formatText } from './formatText';
+import { useTheme } from '../../theme';
+import styles from './styles';
+
+interface IMarkdownPreview {
+	msg?: string;
+	numberOfLines?: number;
+	testID?: string;
+	style?: StyleProp<TextStyle>[];
+}
+
+const MarkdownPreview = ({ msg, numberOfLines = 1, testID, style = [] }: IMarkdownPreview): React.ReactElement | null => {
+	if (!msg) {
+		return null;
+	}
+
+	const { theme } = useTheme();
+
+	let m = formatText(msg);
+
+	// Ex: '[ ](https://open.rocket.chat/group/test?msg=abcdef)  Test'
+	// Return: 'Test'
+	m = m.replace(/^\[([\s]*)\]\(([^)]*)\)\s/, '').trim();
```

Comment:
We reusing this regex on markdown/index.
We should create a separate function.
Very similar to the work we've done to `formatText`.